### ISSUE TITLE
files appearing in the upload box but not being uploaded when using doi or rdf

### ISSUE THREAD

**Issue by esgomezm** [bioimage-io/archive.bioimage.io#315]
Hi hi!

When trying to upload a model indicating the doi to an existing zenodo repo or an rdf yaml, the upload UI shows the files as if they were going to be uploaded. However, when creating the new zenodo repo, this files are not uploaded. I think this is confusing as you expect them to be uploaded. (See the images. I'm using this one: https://zenodo.org/record/7254197/files/rdf.yaml). Could we maybe remove the files from appearing in the upload box?

Also, people are not really checking the zenodo repository before publishing it. Should we be more specific?
![Screenshot 2022-12-01 at 11 43 41](https://user-images.githubusercontent.com/43643518/205044405-f4439cc0-e8a7-402d-bbfb-9d541426cb8b.png)
![Screenshot 2022-12-01 at 11 43 59](https://user-images.githubusercontent.com/43643518/205044455-234faf9b-7413-450f-abad-2a630845a26e.png)
![Screenshot 2022-12-01 at 11 44 25](https://user-images.githubusercontent.com/43643518/205044519-bd0f02b4-33c7-4c09-9780-f8605fce8f15.png)
![Screenshot 2022-12-01 at 11 44 38](https://user-images.githubusercontent.com/43643518/205044558-99c82db4-7b5a-48aa-84e0-37db32f837f9.png)
![Screenshot 2022-12-01 at 11 44 59](https://user-images.githubusercontent.com/43643518/205044612-9e7220a4-72c8-404a-8ed1-3cefdf412f1b.png)

@oeway @FynnBe

**Comment by FynnBe** [comment]
an existing zenodo record should also rather be updated instead of 'cloned'... 
maybe that is where this issue arises from as well--asuming that it is an update for a new version and not a new record.

**Comment by oeway** [comment]
Hi @esgomezm I just tested with the same, but I saw the similar deposit, the red one should give you the "Update as a new version" button if you own this deposit:

<img width="871" alt="Screenshot 2022-12-01 at 14 14 41" src="https://user-images.githubusercontent.com/478667/205170634-4864b958-e7fa-4a66-96a5-234ac217b410.png">

As @FynnBe said, if you start with a doi or rdf url, it assumes that you want to update with a new version.

**Comment by esgomezm** [comment]
Yes, this was an example for you to see the problem. Just changing the title to something random you will see that it does not upload the content of zenodo
